eosdac-material/README.md

```diff
@@ -30,8 +36,23 @@ npm install
 ```
 cp src/statics/config.jungle.json src/statics/config.json
 ```
+*Please note that you may need to change the parameters of the jungle testnet config file periodically.*
```

Comment:
Not sure this is needed in that if we have to change parameters, we should have that committed to our repo already.